package.json

```diff
@@ -13,6 +13,7 @@
     "debug": "npm run build -- --watch --debug",
     "licenses": "license-report --config=./.license-report-config.json > dist/licenses.html",
     "lint": "eslint",
```

Comment:
We can change that to be `lint:src` and create a new `lint` command, which does both - so the test GitHub Action would not change at all.